batch_requests/utils.py

```diff
@@ -101,7 +101,8 @@ def get_wsgi_request_object(curr_request, method, url, headers, body):
 
     data = body
     if data:
-        data = json.dumps(data)
+        if type(data) is dict:
```

Comment:
I think should check for a string instead, since other data types can be json dumped?